src/js/pensions/helpers.jsx

```diff
@@ -61,7 +61,7 @@ export function fileHelp({ formContext }) {
   return (
     <p>
       Files we accept: pdf, jpg, png<br/>
-      Maximum file size: 2MB
+      Maximum file size: 20MB
```

Comment:
I wonder...Would it be worth making this match the `maxSize` in the `formConfig`? That is, pass the `maxSize` in and have it do some calculations and print out the size in MB.

Might be a bit of a pain, but that'd prevent us from displaying the wrong number. Just a precautionary thing.

"It _says_ I can upload a 20MB jpg, but it won't let me upload a 17.5MB one! Ugh!"